pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -153,8 +153,15 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			},
 			Rules: []authorizationapi.PolicyRule{
 				{
-					Verbs:           sets.NewString("get"),
-					NonResourceURLs: sets.NewString("/healthz", "/healthz/*", "/version", "/api", "/oapi", "/osapi", "/api/", "/oapi/", "/osapi/"),
+					Verbs: sets.NewString("get"),
+					NonResourceURLs: sets.NewString(
+						"/healthz", "/healthz/*",
+						"/version",
+						"/api", "/api/", "/api/v1", "/api/v1/",
+						"/apis", "/apis/", "/apis/extensions", "/apis/extensions/", "/apis/extensions/v1beta1", "/apis/extensions/v1beta1/",
+						"/oapi", "/osapi",
+						"/oapi/", "/osapi/", "/oapi/v1", "/oapi/v1/",
```

Comment:
put the /osapi rules on a line together with a comment that we can't remove them for backwards compatibility, even though no API's are surfaced under them any more